docs/reference/testflinger-server-conf.rst

```diff
@@ -37,4 +39,5 @@ Example configuration
   MONGODB_PASSWORD="testflinger"
   MONGODB_DATABASE="testflinger_db"
   MONGODB_HOST="mongo"
-  MONGODB_URI="mongodb://mongo:27017/testflinger_db"
\ No newline at end of file
+  MONGODB_URI="mongodb://mongo:27017/testflinger_db"
+  JWT_SIGNING_KEY="my_secret_key"
```

Comment:
Let's also add this to the example override file in `devel/docker-compose-override.yml`